Yoast/wordpress-seo
# Set OG image not properly outputted

## Summary

This PR can be summarized in the following changelog entry:

* Improves the way that the OpenGraph is determined for front pages, especially in the case of static front pages.

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* Go to SEO → Social → Facebook and select a default image (let's call it Image A)
* Set your homepage to a static page
* Edit the homepage and select a different `og:image` (Image B) through Yoast SEO
* Check the homepage's source, the `og:image` now displays the set image for the frontpage.

## Quality assurance


Fixes #9650
